lib/Jsrt/ChakraCommon.h

```diff
@@ -500,7 +500,12 @@ typedef UINT32 DWORD;
         /// <summary>
         ///     Specified script is internal and non-user code. Hidden from debugger
         /// </summary>
-        JsParseScriptAttributeLibraryCode = 0x1
+        JsParseScriptAttributeLibraryCode = 0x1,
+        /// <summary>
+        ///     ChakraCore assumes ExternalArrayBuffer is Utf8 by default.
+        ///     This one needs to be set for Utf16
```

Comment:
nit: description may need to mention parse script context, otherwise reading by itself is confusing (why ExternalArrayBuffer has anything to do with utf8).